packages/reusable-blocks/src/components/reusable-blocks-menu-items/reusable-blocks-manage-button.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * WordPress dependencies
+ */
+import { MenuItem } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { isReusableBlock } from '@wordpress/blocks';
+import { useSelect } from '@wordpress/data';
+import { BlockSettingsMenuControls } from '@wordpress/block-editor';
+import { addQueryArgs } from '@wordpress/url';
+
+function ReusableBlocksManageButton( { clientId } ) {
+	const { isVisible } = useSelect(
+		( select ) => {
+			const { getBlock } = select( 'core/block-editor' );
+			const { canUser } = select( 'core' );
+			const blockObj = getBlock( clientId );
+
+			const reusableBlock =
+				blockObj && isReusableBlock( blockObj )
+					? select( 'core' ).getEntityRecord(
+							'postType',
+							'wp_block',
+							blockObj.attributes.ref
+					  )
+					: null;
```

Comment:
When I tested, `blockObj.attributes.ref` and `reusableBlock.id` were the same value, so just wondering if this call to `getEntityRecord` is needed? Is this to check that a post actually exists for the editor that's about to be loaded?

Worth noting that while the reusable block is saving I believe it has a temporary id (mentioned here - https://github.com/WordPress/gutenberg/issues/19115), so we might want to check for that.

Reply:
Good point. I don't think `getEntityRecord` is needed here. It was a remnant from the `ReusableBlockDeleteButton` component, but I don't think it is relevant for the user experience in this case. I removed it in my most recent commit.

As for the temporary id, I cannot replicate the ref attribute being anything other than a number id. I looked for documentation on this for a while but came up empty-handed.